k6-operator/Makefile

```
@@ -0,0 +1,28 @@
+.DEFAULT_GOAL := gen-manifests
+
+CHART_VERSION=3.8.0
+APP_VERSION=0.0.16
+
+
+.PHONY: gen-manifests
+gen-manifests:
+	helm repo add grafana https://grafana.github.io/helm-charts
```

```suggestion
	helm repo list | grep grafana || helm repo add grafana https://grafana.github.io/helm-charts
```

Make this action idempotent